flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointMetrics.java

```diff
@@ -59,19 +56,13 @@ public CheckpointMetrics(
 		checkArgument(bytesBufferedInAlignment >= -1);
 		checkArgument(alignmentDurationNanos >= -1);
 
-		this.bytesBufferedInAlignment = bytesBufferedInAlignment;
 		this.alignmentDurationNanos = alignmentDurationNanos;
 		this.syncDurationMillis = syncDurationMillis;
 		this.asyncDurationMillis = asyncDurationMillis;
 	}
 
 	public long getBytesBufferedInAlignment() {
```

Comment:
should we further remove this method because it is always return `0` after this PR? If so how many changes it will be involved?

Reply:
I further removed the relevant code except those in rest handler, the rest handler always return 0 to the web ui, I think we need another ticket to further remove it in web ui.